docs/sources/panels/transformations/config-from-query.md

```diff
@@ -0,0 +1,67 @@
++++
+title = "Config from query results"
+weight = 300
++++
+
+# Config from query results transform
+
+This transformation allow you select one query and from it extract [standard options]({{< relref "./standard-options.md" >}}) like **Min**, **Max**, **Unit** and **Thresholds** and apply it to other query results. This enables dynamic query driven visualization configuration.
+
+If you want to extract a unique config for every row in the query then try the [Rows to fields]({{< relref "./rows-to-fields" >}}) transformation instead.
+
+### Options
+
+- **Config query**: Selet the query that returns the data you want to use as configuration.
+- **Apply to**: Select what fields or series to apply the configuration to.
+- **Apply to options**: Usually a field type or field name regex depending on what option you selected in **Apply to**.
+
+### Field mapping table
+
+Below the configuration listed above you will find the field table. Here all fields found in the data returned by the config query will be listed along with a **Use as** and **Select** option. This table gives you control over what field should be mapped to which config property and if there are multiple rows which value to select.
```

Comment:
Suggestion: `Below the configuration listed above` is a little hard to parse. I tried to break it down into slightly smaller chunks.

What about: 

This transformation includes a field table which lists all fields in the data returned by the config query. This table gives you control over what field should be mapped to each config property. You can choose which value to select if there are multiple rows in the returned data. Each field has both a **Use as** and **Select** option.

I'm not sure if this is accurate or better now that I've written it. :/

Reply:
updated!